cmake/DDCConfig.cmake.in

```diff
@@ -6,10 +6,25 @@
 
 include(CMakeFindDependencyMacro)
 
+set(DDC_BUILD_DOUBLE_PRECISION @DDC_BUILD_DOUBLE_PRECISION@)
+set(DDC_BUILD_KERNELS_FFT @DDC_BUILD_KERNELS_FFT@)
+set(DDC_BUILD_KERNELS_SPLINES @DDC_BUILD_KERNELS_SPLINES@)
+set(DDC_BUILD_PDI_WRAPPER @DDC_BUILD_PDI_WRAPPER@)
+
+find_package(Kokkos 4.4...4.5 CONFIG)
```

Comment:
```suggestion
find_dependency(Kokkos 4.4...4.5 CONFIG)
```

Reply:
Why forcing "CONFIG"?

Reply:
Not sure why it is there, I think we needed CONFIG for some package and it has copy-pasted. I can remove it if you prefer

Reply:
done in 47ada334af847d5288ab2366adb9f1e56dab2e33